docs/workloads/dependencies/images.md

```diff
@@ -44,6 +44,25 @@ RUN pip install --no-cache-dir pandas \
     && conda clean -a
 ```
 
+If you need to upgrade the Python Runtime version from its current 3.6.9 on your custom image using the existing conda environment, follow this procedure:
```

Comment:
Do you think we should also mention how to update the python version when we talk about `dependencies.sh` (in `system-packages.md`)?